test/script/swslither.js

```diff
@@ -0,0 +1,154 @@
+/* swslither.js */
+
+ui.debug.addDebugData("swslither", {
+	url: "5/5/2c5a2a5c2a136a3c6a3",
+	failcheck: [
+		[
+			"brNoLine",
+			"pzprv3.1/swslither/5/5/2 . . . 5 /. 2 . 5 . /. . 2 . 1 /3 6 . 3 . /. . 6 . 3 /0 0 0 0 0 /0 0 0 0 0 /0 0 0 0 0 /0 0 0 0 0 /0 0 0 0 0 /0 0 0 0 0 0 /0 0 0 0 0 0 /0 0 0 0 0 0 /0 0 0 0 0 0 /0 0 0 0 0 0 /0 0 0 0 0 /0 0 0 0 0 /0 0 0 0 0 /0 0 0 0 0 /0 0 0 0 0 /0 0 0 0 0 /"
+		],
+		[
+			"lnBranch",
+			"pzprv3.1/swslither/5/5/2 . . . 5 /. 2 . 5 . /. . 2 . 1 /3 6 . 3 . /. . 6 . 3 /0 0 0 0 0 /0 0 0 0 0 /0 0 0 0 0 /0 0 0 0 0 /0 0 0 0 0 /1 0 0 0 0 0 /1 1 0 0 0 0 /1 0 0 0 0 0 /1 0 0 0 0 0 /1 0 0 0 0 0 /1 1 0 0 0 /0 1 0 0 0 /0 0 0 0 0 /1 0 0 0 0 /0 0 0 0 0 /0 0 0 0 0 /"
+		],
+		[
+			"lnCross",
+			"pzprv3.1/swslither/5/5/2 . . . 5 /. 2 . 5 . /. . 2 . 1 /3 6 . 3 . /. . 6 . 3 /0 0 0 0 0 /0 0 0 0 0 /0 0 0 0 0 /0 0 0 0 0 /0 0 0 0 0 /1 0 0 0 0 0 /1 1 0 1 0 0 /1 1 1 0 1 0 /1 1 1 0 1 0 /0 0 0 0 1 0 /1 1 0 0 0 /0 1 1 0 0 /0 0 1 0 0 /0 0 0 0 0 /1 0 1 1 1 /0 0 0 0 0 /"
+		],
+		[
+			"nmLineNe",
+			"pzprv3.1/swslither/5/5/2 . . . 5 /. 2 . 5 . /. . 2 . 1 /3 6 . 3 . /. . 6 . 3 /0 0 0 0 0 /0 0 0 0 0 /0 0 0 0 0 /0 0 0 0 0 /0 0 0 0 0 /1 0 0 0 0 0 /1 1 0 1 0 0 /1 1 1 0 0 0 /1 1 1 0 0 0 /0 0 0 0 0 0 /1 1 0 0 0 /0 1 1 0 0 /0 0 1 0 0 /0 0 0 0 0 /1 0 1 1 0 /0 0 0 0 0 /"
+		],
+		[
+			"lnPlLoop",
+			"pzprv3.1/swslither/5/5/2 . . . 5 /. 2 . 5 . /. . 2 . 1 /3 6 . 3 . /. . 6 . 3 /0 0 0 0 0 /0 0 0 0 0 /0 0 0 0 0 /0 0 0 0 0 /0 0 0 0 0 /1 0 1 1 0 1 /1 1 0 1 0 1 /1 1 1 0 0 1 /1 1 1 1 1 1 /0 0 0 0 1 1 /1 1 0 1 1 /0 1 0 0 0 /0 0 1 0 0 /0 0 0 1 0 /1 0 1 0 0 /0 0 0 0 1 /"
+		],
+		[
+			"lnDeadEnd",
+			"pzprv3.1/swslither/5/5/2 . . . 5 /. 2 . 5 . /. . 2 . 1 /3 6 . 3 . /. . 6 . 3 /0 0 0 0 0 /0 0 0 0 0 /0 0 0 0 0 /0 0 0 0 0 /0 0 0 0 0 /1 0 0 0 0 0 /1 1 0 1 0 0 /1 1 1 0 0 1 /1 1 1 1 1 1 /0 0 0 0 1 1 /1 1 1 1 0 /0 1 1 0 0 /0 0 1 0 0 /0 0 0 1 0 /1 0 1 0 0 /0 0 0 0 1 /"
+		],
+		[
+			"sheepOutsideOrWolfInside",
+			"pzprv3.1/swslither/5/5/2 . . . 5 /. 2 . 5 . /. . 2 . 1 /3 6 . 3 . /. . 6 . 3 /0 0 0 0 0 /0 0 0 0 0 /0 0 0 0 0 /0 0 0 0 0 /0 0 0 0 0 /1 0 0 0 0 1 /1 1 0 0 1 1 /0 0 1 0 0 1 /1 0 0 1 1 1 /0 0 0 0 1 1 /1 1 1 1 1 /0 1 1 1 0 /1 0 1 1 0 /1 1 0 1 0 /1 1 1 0 0 /0 0 0 0 1 /"
+		],
+		[
+			null,
+			"pzprv3.1/swslither/5/5/2 . . . 5 /. 2 . 5 . /. . 2 . 1 /3 6 . 3 . /. . 6 . 3 /0 0 0 0 0 /0 0 0 0 0 /0 0 0 0 0 /0 0 0 0 0 /0 0 0 0 0 /1 0 0 0 0 1 /1 1 0 1 0 1 /1 1 1 0 0 1 /1 1 1 1 1 1 /0 0 0 0 1 1 /1 1 1 1 1 /0 1 1 0 0 /0 0 1 0 0 /0 0 0 1 0 /1 0 1 0 0 /0 0 0 0 1 /"
+		]
+	],
+	inputs: [
+		/* 回答入力テスト */
+		{ input: ["newboard,5,1", "playmode"] },
```

Comment:
Most of the input tests are already covered by `slither`, so everything except the sheep/wolf inputs can be removed here

Reply:
Makes sense, done.